sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/fluent/PolicyDefinitionsClient.java

```diff
@@ -299,150 +299,150 @@ Mono<PolicyDefinitionInner> createOrUpdateAtManagementGroupAsync(String policyDe
      * @return the policy definition along with {@link Response}.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    Response<PolicyDefinitionInner> createOrUpdateAtManagementGroupWithResponse(String policyDefinitionName,
-        String managementGroupId, PolicyDefinitionInner parameters, Context context);
+    Response<PolicyDefinitionInner> createOrUpdateAtManagementGroupWithResponse(String managementGroupId,
+        String policyDefinitionName, PolicyDefinitionInner parameters, Context context);
 
     /**
      * Creates or updates a policy definition in a management group.
      * 
      * This operation creates or updates a policy definition in the given management group with the given name.
      * 
-     * @param policyDefinitionName The name of the policy definition to create.
      * @param managementGroupId The ID of the management group.
+     * @param policyDefinitionName The name of the policy definition to create.
      * @param parameters The policy definition properties.
      * @throws IllegalArgumentException thrown if parameters fail the validation.
      * @throws com.azure.core.management.exception.ManagementException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
      * @return the policy definition.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    PolicyDefinitionInner createOrUpdateAtManagementGroup(String policyDefinitionName, String managementGroupId,
+    PolicyDefinitionInner createOrUpdateAtManagementGroup(String managementGroupId, String policyDefinitionName,
         PolicyDefinitionInner parameters);
 
     /**
      * Deletes a policy definition in a management group.
      * 
      * This operation deletes the policy definition in the given management group with the given name.
      * 
-     * @param policyDefinitionName The name of the policy definition to delete.
      * @param managementGroupId The ID of the management group.
+     * @param policyDefinitionName The name of the policy definition to delete.
      * @throws IllegalArgumentException thrown if parameters fail the validation.
      * @throws com.azure.core.management.exception.ManagementException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
      * @return the {@link Response} on successful completion of {@link Mono}.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    Mono<Response<Void>> deleteAtManagementGroupWithResponseAsync(String policyDefinitionName,
-        String managementGroupId);
+    Mono<Response<Void>> deleteAtManagementGroupWithResponseAsync(String managementGroupId,
+        String policyDefinitionName);
```

Comment:
Is this expected? Seems a breaking change to me...

Reply:
they did swtiched the order
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/resources/resource-manager/Microsoft.Authorization/stable/2021-06-01/policyDefinitions.json#L282-L296
to
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/resources/resource-manager/Microsoft.Authorization/stable/2023-04-01/policyDefinitions.json#L287-L302

from the path, later order does make more sense (but yes, it breaks if user uses this API)